tap_bing_ads/__init__.py

```diff
@@ -136,6 +147,9 @@ def set_options(self, **kwargs):
         self._options = kwargs
         kwargs = ServiceClient._ensemble_header(self.authorization_data, **self._options)
         kwargs['headers']['User-Agent'] = get_user_agent()
+        request_timeout = get_request_timeout()
+        # setting the timeout parameter using the set_options which sets timeout in the _soap_client
+        kwargs['timeout'] = request_timeout
```

Comment:
No need to define the `request_timeout` variable, you can write it like below: 
 ```
# setting the timeout parameter using the set_options which sets timeout in the _soap_client
        kwargs['timeout'] = get_request_timeout()
```

Reply:
done